tests/e2e/playwright.side-effects.config.ts

```diff
@@ -21,11 +21,17 @@ export default defineConfig( {
 		{
 			name: 'blockThemeConfiguration',
 			testMatch: /block-theme.setup.ts/,
+			dependencies: [ 'authSetup' ],
 		},
 		{
 			name: 'blockThemeWithGlobalSideEffects',
 			testMatch: /.*.block_theme.side_effects.spec.ts/,
-			dependencies: [ 'blockThemeConfiguration' ],
+			dependencies: [ 'blockThemeConfiguration', 'authSetup' ],
```

Comment:
Don't think this is necessary because `blockThemeConfiguration` depends on `authSetup`